src/components/quoteSlider/QuoteSlider.tsx

```diff
@@ -0,0 +1,145 @@
+import React, { useState, useRef, useEffect } from 'react';
+import styled from '@emotion/styled';
+import Image from 'next/image';
+import LeftArrow from '../../../public/leftArrow.png';
+import RigthArrow from '../../../public/rightArrow.png';
+import { Quote } from './Quote';
+
+export type QuoteT = {
+  englishQuote: string;
+  englishCite: string;
+  koreanQuote: string;
+  koreanCite: string;
+};
+interface Props {
+  quotes: QuoteT[];
+}
+
+export const QuoteSlider = ({ quotes }: Props) => {
+  const slideRef = useRef<HTMLDivElement>(null);
+  const [currentSlide, setCurrentSlide] = useState(0);
+  const TOTAL_SLIDES = quotes.length - 1;
+  const SLIDE_BOX_WIDTH = 1200 * TOTAL_SLIDES;
+
+  const NextSlide = () => {
+    if (currentSlide >= TOTAL_SLIDES) {
+      return;
+    } else {
+      setCurrentSlide(currentSlide + 1);
+    }
```

Comment:
`Guard-Clause` 를 검색해보시면 좋을 것 같아요.

if-else문 & if 중첩문을 clean하게 만들어 주는 마법을 부려요~

early return 도 같이 찾아보세요 ㅎㅎ

```suggestion
  const NextSlide = () => {
    if (currentSlide >= TOTAL_SLIDES) {
      return;
    }
    
    setCurrentSlide(currentSlide + 1);
```

Reply:
Guard-Clause라는건 처음 들어봤습니다 🫠  감사합니다!!